site/404.html

```diff
@@ -1,11 +1,18 @@
-{% extends "_layouts/default.html" %}
```

Comment:
In most of our projects, these templates live in a `templates` folder. I get that vanilla-framework is different in that it's not in and of itself a website project, but I'm not sure `templates` is a much worse name than `site`. Don't you think it would probably be okay to simply call it `templates` for consistency?

Reply:
I don't mind, if we are calling it `templates` in others projects we should do the same. WDYT @bartaz ?

Reply:
Yes, I added it to todos, let's do it separately to avoid too moving files again that will cause conflicts with Lyubos design changes.